src/scss/grommet-core/_objects.accordion.scss

```diff
@@ -0,0 +1,10 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+.grommet .#{$grommet-namespace}accordion {
```

Comment:
I believe we don't need the `.grommet` here. We only need that for layers, eg: Menu drop, Layer...